contracts/storage/AssertionStorage.sol

```diff
@@ -1,31 +1,21 @@
 // SPDX-License-Identifier: MIT
 
-pragma solidity ^0.8.4;
+pragma solidity ^0.8.16;
```

Comment:
Why did we change storage contracts? Does this means that we need to redeploy them also?

Reply:
I don't think that it's smart to change storage contract and leave them undeployed. This means that the contract that is deployed is different than the one we have in our codebase. Also the ABI is different. I know that this doesn't affect functionality, but i think that in couple of months we will forget this and we will look at these contracts and we will think that they are deployed with this code.

Reply:
I would say that it makes sense to revert changes in code for the constructor and similar changes, but regarding the version, as we deployed them with 0.8.16 to me it looks more correct to have that info also on contracts. Regarding ABI, as I understood we added some change to enhance the ABI and make it easier to build clients, but nothing functional should have changed?